### PR TITLE
Pin redis and nginx versions

### PR DESCRIPTION
**Pull request by noisecapella**
#### Pre-Flight checklist

- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested

#### What are the relevant tickets?
https://trello.com/c/1KnhFWrP/64-anonymous-access-to-user-profiles

#### What's this PR do?
Pin a couple docker container versions so that everybody is on the same version

#### How should this be manually tested?
Site should load

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ services:
       - "5432"
 
   redis:
-    image: redis
+    image: redis:3.0.3
```
Comment (aliceriot) [inline docker-compose.yml]: is this the redis version we're using on prod? I just checked and I have 5.0.7 installed locally, and this version doesn't seem to work properly on my laptop

Comment (aliceriot) [reply docker-compose.yml]: 3.0.3 is also [almost five years old](https://github.com/antirez/redis/releases/tag/3.0.3) as far as I can tell, so we may want to shoot for a newer version

Comment (noisecapella) [reply docker-compose.yml]: Yes, I'll use 5.0.7. I think I had used the version that micromasters was pinned to

**Comment by noisecapella** [comment]
@aliceriot Ready for review again

**Pull request merged**